docs/Reference/CLI/Subcommands/Slashing-Protection.md

```diff
@@ -146,7 +146,186 @@ The file to export the slashing protection database to.
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --data-base-path=<PATH>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --data-base-path=/home/me/me_node
+    ```
+
+The path to the Teku data directory. The default directory is OS dependent:
+
+* macOS: `~/Library/teku`
+* Unix/Linux: `$XDG_DATA_HOME/teku` if `$XDG_DATA_HOME` is set; otherwise `~/.local/share/teku`
+* Windows: `%localappdata%\teku`.
+
+The default Docker image location is `/root/.local/share/teku`.
+
+### data-validator-path
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --data-validator-path=<PATH>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --data-validator-path=/home/me/me_validator
+    ```
+
+Path to validator client data. Defaults to `<data-base-path>/validator` where `<data-base-path>`
+is specified using [`--data-base-path`](#data-base-path-data-path).
+
+### eth1-deposit-contract-address
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --eth1-deposit-contract-address=<ADDRESS>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --eth1-deposit-contract-address=0x77f7bED277449F51505a4C54550B074030d989bC
+    ```
+
+Ethereum 1.0 address of the deposit contract. Only required when creating a custom network.
+
+### initial-state
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --initial-state=<STRING>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --initial-state=/home/me/genesis.ssz
+    ```
+
+Path or URL to an SSZ-encoded state file. The state file can be used to specify the genesis state,
+or a [recent finalized checkpoint state from which to sync].
+
+This option does not need to be specified if the initial state is provided by the network specified
+using the [`--network`](#network) option.
+
+### network
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --network=<NETWORK>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --network=mainnet
+    ```
+
+Predefined network configuration. Accepts a predefined network name, or file path or URL
+to a YAML configuration file. The default is `mainnet`.
+
+Possible values are:
+
+| Network   | Chain   | Type        | Description                                      |
+|-----------|---------|-------------|--------------------------------------------------|
+| `mainnet` | Eth 2.0 | Production  | Main network.                                    |
+| `minimal` | Eth 2.0 | Test        | Used for local testing and development networks. |
+| `pyrmont` | Eth 2.0 | Test        | Multi-client testnet.                            |
+| `prater`  | Eth 2.0 | Test        | Multi-client testnet.                            |
+
+Predefined networks can provide defaults such the initial state of the network,
+bootnodes, and the address of the Ethereum 1.0 deposit contract.
```

Comment:
network shouldn't have any effect either.

Reply:
i realised it will actually be fairly important if config-file isn't specified - sorry to mis-inform you on the network parameter...